content/src/view.js

```diff
@@ -250,11 +255,13 @@ function setOnCallback(tag, cb) {
 
 function fireEvent(tag, args) {
   if (tag in state.callbacks) {
-    var cb = state.callbacks[tag];
-    if (cb) {
-      cb.apply(null, args);
-    }
-  }
+     var cbs = state.callbacks[tag].slice();
```

Comment:
There is an extra space before `var cbs = state.callbacks[tag].slice();` :stuck_out_tongue: